selenium/che-selenium-test/src/test/resources/projects/nodejs-with-yaml/deployment.yaml

```diff
@@ -134,7 +134,7 @@ spec:
                   name: openshift-identity-provider
                   optional: true
             - name: CHE_WORKSPACE_PLUGIN__REGISTRY__URL
-              value: 'NULL'
+              value: 'https://che-plugin-registry.openshift.io'
```

Comment:
I believe this yaml file is not used for deploying Che on the OpenShift. I think it doesn't make sense to update it. It would be even better to replace this Che Deployment Config yaml with simpler yaml which is needed for `nodejs-with-yaml` testing 
@eclipse/eclipse-che-qa Please correct me if I'm wrong and this test requires actual Che Server configuration.